changes/2584.fix.md

```diff
@@ -0,0 +1 @@
+The VFolder lookup API endpoint `GET /func/folders/{folderName}` now correctly returns `null` for `user` and `group` fields when they are absent, instead of returning the string "None".
```

Comment:
```suggestion
The VFolder lookup API `GET /func/folders/{folderName}` now returns `null` for missing `user` and `group` fields, instead of returning `None` as string.
```

Reply:
Let's refine the changelog. The others look good to me.